recipes/pro-mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "pro-mdnsd"
+    license = "BSD-3-Clause"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://github.com/Pro/mdnsd"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Improved version of Jeremie Miller's MDNS-SD implementation"
+    topics = ("dns", "daemon", "multicast", "embedded", "c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "compile_as_cpp": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "compile_as_cpp": False
+    }
+    generators = "cmake", "cmake_find_package"
```

Comment:
```suggestion
    generators = "cmake"
```

Not needed, we don't have requirements here

Reply:
this will be resolved when 0.8.2 is released

Reply:
so please, add it only when 0.8.2 be available.